src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -2960,6 +2963,31 @@ namespace SimAirServingZones {
                     // not converge
                     ControllerConvergedFlag = PrimaryAirSystem(AirLoopNum).ControlConverged(AirLoopControlNum);
                     IsUpToDateFlag = true;
+                } else {
```

Comment:
This will require multiple more iterations of the air loop components. Wouldn't it be better to use the lesser of T and TdpFw as the temperature set point up at line 1283 - 1289 of HVACControllers?

Reply:
I can try that on a separate branch, but I think it will over-dehumidify.

Reply:
Or use T and TdpFw as a trigger to choose which set point to use, temperature or humidity ratio.

Reply:
No, don't need to try that. The humidity ratio setpoint is 0.011388 which is roughly Tdp=15.9C. There are many hours where the coil can meet the humidity ratio setpoint with a leaving temperature of 16-17C, and there are many hours where the leaving air is saturated. That's why this needs to switch gears and can't just use Tdp.

Reply:
Hmm, how can Tdp come out lower than supply air T? If a dry coil then yes, but a wet coil? Not your problem at this point.

Reply:
'cause the water temp is well below Tdp, and some of the air bypasses the coil, so the resulting mix can be drier than saturation.